src/lightbox/LightboxActionSheet.js

```diff
@@ -32,6 +32,12 @@ type ButtonType = {
   onPress: (props: ButtonProps) => void | boolean | Promise<any>,
 };
 
+type ActionSheetOptionsType = {
+  showActionSheetWithOptions: (Object, (number) => void) => void,
```

Comment:
prefer to add methods at end in the list.

Reply:
Okay :+1: . I'll remember next time.

Reply:
Changed this :+1: